src/grouping.jl

```diff
@@ -32,3 +32,9 @@ Base.getproperty(g::Grouping, prop::Symbol) = prop == :levels ? nothing : getfie
 function StatsModels.modelcols(::CategoricalTerm{Grouping}, d::NamedTuple)
     return error("can't create model columns directly from a Grouping term")
 end
+
+function StatsModels.ContrastsMatrix(
+    contrasts::Grouping, levels::AbstractVector{T}
+) where {T}
```

Comment:
type param no longer actually used...

```suggestion
function StatsModels.ContrastsMatrix(
    contrasts::Grouping, levels::AbstractVector
)
```

Reply:
I left it in there to make sure we're more specific than the methods in StatsModels, but I'll test leaving it out.

Reply:
I _think_ it's teh same (union type either way) but better to make sure.